File: 283. Move Zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int ptr = 0;
        for(auto& it: nums){
            if(it!=0){
                nums[ptr] = it;
                ptr++;
            }   
        }
        
        for(int i=ptr;i<nums.size();i++)
            nums[i]=0;
        
    }
};

File: 283. Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        ptr1 = 0
        for num in nums:
            if num:
                nums[ptr1] = num
                ptr1 += 1
        for i in range(ptr1, len(nums)):
            nums[i] = 0

